openlattice.yaml

```diff
@@ -1739,7 +1739,8 @@ paths:
                       additionalProperties:
                         type: array
                         items:
-                          type: object
+                          type: string
+                          format: uuid
```

Comment:
Isn't this supposed to represent the type of the values in the map?

Reply:
Oh are these the data values?  Or entityKeyIds?

Reply:
It returns the data values: `Map<UUID, Map<UUID, Map<UUID, Map<FullQualifiedName, Set<Object>>>>>`

Reply:
But what are the last objects in `Set<Object>`?  Strings or UUIDs?

Reply:
They can be anything (depending on the mapping property type), that's why the type is an Object. I think in these cases we use string, right?

Reply:
Ok, back to string :) 

Yeah, it seems to me that in openapi-specs, an object is always a map, and should always be followed by the keyword `properties` to indicate the key/value-pairs.
These are the primitive datatypes currently accepted: https://swagger.io/specification/#dataTypes

We could put in an list of options as return-bodies, but yeah, so far I've always just put in strings...

Reply:
Shouldn't the type be object then? Since you could theoretically get a return value of `4` and not `"4"`

Reply:
Myeah, I've noticed R is interpreting `object` as `map` and things get ... weird.  And I can't find any examples in the documentation on `object` being used not as map. 

If anything we should probably say `any of the primitive datatypes` (link above), but that would need to be an overhaul throughout this document so outside the scope of this PR.

Reply:
Actually, to make everything make even more sense: an `object` is interpreted as `map` in R, where these are called `lists`, not to be confused with `vectors` in R that are called `lists` everywhere else.
👌

Reply:
Interesting, so object can't be an array or list? i.e a json array is valid primitive json object.

Reply:
Hm, then maybe it's a R-generator issue that isn't able to deserialize and transform into R-objects.  But then please let's keep it the way it is, unless someone wants to go fix the R-generator ;-)